modules/cluster-logging-collector-log-forward-loki.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forward-loki_{context}"]
+= Forwarding logs to Grafana Loki
+
+You can forward logs to an external Loki logging system in addition to, or instead of, the internal default {product-title} Elasticsearch instance.
+
+To configure log forwarding to Grafana Loki, create a `ClusterLogForwarder` custom resource (CR) with an output to Loki and a pipeline that uses the output. The output to Loki can use the HTTP (insecure) or HTTPS (secure HTTP) connection.
+
+.Prerequisites
+
+* You must have an Loki logging system that is configured to receive the logging data using the specified protocol or format.
+
+.Procedure
+
+. Create a `ClusterLogForwarder` CR YAML file similar to the following:
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance <1>
+  namespace: openshift-logging <2>
+spec:
+  outputs:
+   - name: loki-insecure <3>
+     type: "loki" <4>
+     url: http://loki.insecure.com:9200 <5>
+   - name: loki-secure
+     type: "loki"
+     url: https://loki.secure.com:9200 <6>
+     secret:
+        name: loki-secret <7>
+  pipelines:
+   - name: application-logs <8>
+     inputRefs: <9>
+     - application
+     - audit
+     outputRefs:
+     - loki-secure <10>
+     labels:
```

Comment:
I don't think we should overload the loki example with features unrelated to loki - pipeline labels can be used with any output type and are hopefully documented elsewhere?

Reply:
Updated. I removed everything below `- loki-secure <10>`. PTAL to make sure this looks good to you @alanconway